apps/admin-gui/src/app/vos/pages/vo-detail-page/vo-attributes/vo-attributes.component.ts

```diff
@@ -53,9 +53,13 @@ export class VoSettingsAttributesComponent implements OnInit {
   tableId = TABLE_ATTRIBUTES_SETTINGS;
   pageSize: number;
 
+  createAuth: boolean;
+  deleteAuth: boolean;
+  editAuth: boolean;
```

Comment:
I hope this piece of code will make sense when it will be rebased or merge together with commit member:authorization?